README.prod.md

```diff
@@ -37,35 +37,31 @@ Installing Zulip in production
 These instructions should be followed as root.
 
 (1) Install the SSL certificates for your machine to
-  /etc/ssl/private/zulip.key
-  and
-  /etc/ssl/certs/zulip.combined-chain.crt
+    `/etc/ssl/private/zulip.key` and `/etc/ssl/certs/zulip.combined-chain.crt`
```

Comment:
Just merged an extension here explaining how to create a self-signed cert that you'll want to resolve merge conflicts with.